tests/test_logic.py

```diff
@@ -494,3 +494,49 @@ def test_isclose(dtype, rtol, atol):
     np_res = numpy.isclose(a, b, 1e-05, 1e-08)
     dpnp_res = dpnp.isclose(dpnp_a, dpnp_b, rtol, atol)
     assert_allclose(dpnp_res, np_res)
+
+
+@pytest.mark.parametrize("a", [numpy.array([1, 2]), numpy.array([1, 1])])
+@pytest.mark.parametrize(
+    "b",
+    [
+        numpy.array([1, 2]),
+        numpy.array([1, 2, 3]),
+        numpy.array([3, 4]),
+        numpy.array([1, 3]),
+        numpy.array([1]),
+        numpy.array([[1], [1]]),
+        numpy.array([2]),
+        numpy.array([[1], [2]]),
+        numpy.array([[1, 2, 3], [4, 5, 6], [7, 8, 9]]),
+    ],
+)
+def test_array_equiv(a, b):
+    result = dpnp.array_equiv(dpnp.array(a), dpnp.array(b))
```

Comment:
Please add testing with different dtypes for at least one scenario per new function